packages/angular_devkit/core/src/json/schema/interface.ts

```diff
@@ -69,8 +69,7 @@ export interface SchemaValidator {
 
 export interface SchemaFormatter {
   readonly async: boolean;
-  // tslint:disable-next-line:no-any
-  validate(data: any): boolean | Observable<boolean>;
+  validate(data: unknown): boolean | Observable<boolean>;
```

Comment:
This is a breaking change since the interface is public and `unknown` is more restrictive than `any`.

Reply:
Hey @clydin sorry cannot understand what I need to change? can you explain, please

Reply:
Can you revert this one change? The any usage will unfortunately need to stay in this case since it may break users of the public API.  This change could potentially be made in the future but it would need to be in a major release.